web-console/4.0-designs/help-menu/help-menu.md

```diff
@@ -0,0 +1,29 @@
+# Help Menu
+
+The help menu will be accessible at all times from the primary masthead. Users can access it by clicking the help menu dropdown (fa-question-circle-o).
+
+**Menu Dropdown**
+![menu](img/menu.png)
+
+* The menu dropdown will include links to Documentation, the Quick Start Guide, the Command Line Tools page, and the About modal
+  * Documentation will redirect users to an external link
+  * Quick Start Guide will take users to the Quick Start Guide landing page
+  * Command Line Tools will redirect users to the CLI page
+  * About will open the About modal
+
+
+
+**Quick Start Guide**
+![Quick Start Guide](img/quick-start-guide.png)
+
+* The Quick Start Guide landing page will have the quick start guide info as well as information regarding documentation and additional support links
```

Comment:
Seems like we should be linking to the [Quick Start documentation](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/quick-start-guide/quick-start-guide) here if someone wanted more information.  Also just realized it's a little confusing as I went to the Quick Start documentation since we say it appears on the Overview page, but then show it in context of the Status page. Is this on purpose?